Part 2

elabcius5 opened the pull request:
For this PR, I created a client for consuming the service from Octokit API, I associated a model acording to JSON and then I consumed the service in Activity. For now, I log the data in console debug

elabcius5 committed:
Second PR

For this PR, I created a client for consuming the service from Octokit
API, I associated a model acording to JSON and then I consumed the
service in Activity. For now, I log the data in console debug

lucaslabs commented on app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java:
```diff
@@ -0,0 +1,180 @@
+ */
+public class SubscribersModel {
+
+    @SerializedName("login")
```
No need to use `SerializedName` when the name on the JSON field is the same of the attribute.

lucaslabs commented on app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java:
```diff
@@ -0,0 +1,180 @@
+    private String login;
+    @SerializedName("id")
+    private String id;
+    @SerializedName("avatar_url")
```
This case is where `SerializedName` is valid.

lucaslabs commented on app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java:
```diff
@@ -0,0 +1,180 @@
+/**
+ * Created by juan.herrera@globant.com on 11/08/2016.
+ */
+public class SubscribersModel {
```
This model can be immutable: final class, final attributes, NO setters.

lucaslabs commented:
@elabcius5 please add a video that showcase the app running with this changes :smiley:

elabcius5 committed:
Changes

I changed de class, I added the keyword FINAL and I changed the
attributes modifiers

elabcius5 commented:
The video was upload on YouTube. The url is: https://www.youtube.com/watch?v=ASWHeyzsVgM

lucaslabs commented on app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java:
```diff
@@ -5,176 +5,128 @@
-    private String login;
-    @SerializedName("id")
-    private String id;
+    final String login;
```
Private access modifier must be here, this is, `private final String login`. Do the same for the rest of the fields/attributes.

elabcius5 committed:
Correction

I changed the attributes visibility for private

juliocatano commented:
please merge master into this branch, since your first PR was already merged because I continue to see the change for the .gitignore

juliocatano commented:
**LG** after small comment

elabcius5 committed:
Merge branch 'master' into SubscribersSearch_SecondPR
